app/assets/javascripts/miq_application.js

```diff
@@ -1405,7 +1405,12 @@ function miqToolbarOnClick(e) {
     if (typeof params == "undefined") {
       miqJqueryRequest(tb_url, {beforeSend: true, complete: true});
     } else {
-      miqJqueryRequest(tb_url, {beforeSend: true, complete: true, data: params});
+      options = {beforeSend: true, data: params};
```

Comment:
@himdel, not sure how I originally missed that ```var```, good catch.

The refactoring makes sense and I haven't found anything that blows up with ```{data: undefined}```.